scripts/apply_deposits.js

```diff
@@ -2,7 +2,11 @@ const SnappBase = artifacts.require("SnappBase")
 
 module.exports = async (callback) => {
   try {
-    const arguments = await process.argv.slice(4)
+    let sliceBy = 4
```

Comment:
I'd much prefer if we can filter out the parameters that are bothering use (which I assume is `--network`, whatever comes after, and maybe the first few (node, truffle, exec).

It would also allow us to not repeat this code all over (but with different indices), and instead have a utility function that we can use in future scripts as well and that is more prone to changes in the parameter list.